sources/dicom/large_image_source_dicom/__init__.py

```diff
@@ -129,10 +134,12 @@ def __init__(self, path, **kwargs):
             self._largeImagePath = path
         _lazyImport()
         try:
-            self._dicom = wsidicom.WsiDicom.open(self._largeImagePath)
+            self._dicom = self._open_wsi_dicom(self._largeImagePath)
         except Exception:
-            msg = 'File cannot be opened via dicom tile source.'
+            full_e = traceback.format_exc()
```

Comment:
Rather than importing traceback, it is probably sufficient to do
```
except Exception as exc:
  msg = f'File cannot be opened via dicom tile source ({exc}).'
```

Reply:
Fixed!